README.md

```diff
@@ -1,119 +1,118 @@
+- [International Data Spaces](https://www.internationaldataspaces.org)
+- The [GAIA-X](https://www.data-infrastructure.eu) project
 
-``` git clone https://github.com/microsoft/Data-Appliance-GX ```
+_Note: items marked with [TBW] indicate that the respective documentation is yet to be written_
 
-``` cd Data-Appliance-GX ```
+## Built with
 
-```./gradlew clean shadowJar```
+One of the guiding principles in developing the connector is simplicity and keeping the core small and efficient with as
+little external dependencies as possible to avoid version conflicts. We do not want to force any third-party
+dependencies onto our users, so we aim to avoid any of the big frameworks. The connector is a plain Java application
+built with Gradle.
 
-To launch the runtime and client from the root build directory, respectively:
+# Getting Started
 
-```java -jar runtime/build/libs/edc-runtime.jar```
+## Checkout and build code
 
-```java -jar client/build/libs/edc-client.jar```
+The project requires JDK 11+. To get started:
 
-# Build Profiles
+``` shell 
+git clone git@github.com:eclipse-dataspaceconnector/DataSpaceConnector.git
 
-The runtime can be configured with custom modules be enabling various build profiles.
+cd DataSpaceConnector ```
 
-By default, no vault is configured. To build with the file system vault, enable the security profile:
+./gradlew clean build
+```
 
-```./gradlew -Dsecurity.type=fs clean shadowJar ```
+That will build the connector and run unit tests.
 
-The runtime can then be started from the root clone directory using:
+## Run your first connector
 
-``` java -Dedc.vault=secrets/edc-vault.properties -Dedc.keystore=secrets/edc-test-keystore.jks -Dedc.keystore.password=test123 -jar runtime/build/libs/edc-runtime.jar ```
+Connectors can be started using the concept of "launchers", which are essentially compositions of Java modules defined
+as gradle build files. There is a `basic` launcher, which launches a simple connector that has no cloud-based extensions
+whatsoever.
 
-Note the secrets directory referenced above is configured to be ignored. A test key store and vault must be added (or
-the launch command modified to point to different locations). Also, set the keystore password accordingly.
+In a shell run
 
-# A word on launchers
+```shell
+./gradlew :launchers:basic:shadowJar
+java -jar launchers/basic/build/libs/dataspaceconnector-basic.jar
+```
 
-The code base is organized in many different modules, some of which are grouped together using so-called "feature
-bundles". For example, accessing IDS requires a total of 4 modules, which are grouped together in the `ids` feature
-bundle. So developers wanting to use that feature only need to reference `ids` instead all of the 4 modules
-individually. This allows for a flexible and easy composition of the runtime. We'll call those compositions "
-launchers".
+Once it says `"Dataspace Connector ready"` the connector is up and running.
```

Comment:
Maybe not for this PR but would be good to have some examples of queries to interact with the connector (a Swagger spec would be ideal)

Reply:
as of now there is no "default" way to interact with connector. everyone can build their own, as in: you could create a rest api for it.
The connector is essentially just the core and interfaces to extend it.